Cubical/Cohomology/EilenbergMacLane/Rings/RP2wedgeS1.agda

```diff
@@ -0,0 +1,581 @@
+    e-sect = DS-Ind-Prop.f _ _ _ _ (λ _ → isSetH* _ _)
+             refl
+             e-sect-base
+             λ {U V} ind-U ind-V → cong₂ _+H*_ ind-U ind-V
+
+  -----------------------------------------------------------------------------
+  -- Retraction
+
+    e-retr-base : (v : Vec ℕ 2) → (a : fst ℤ/2) →
+                  H*-RP²⋁S¹→ℤ/2[x,y]/<y³,xy,x²> (ℤ/2[x,y]/<y³,xy,x²>→H*-RP²⋁S¹ [ base v a ]) ≡ [ base v a ]
+    e-retr-base (zero ∷ zero ∷ []) a = cong [_] (cong (baseP (0 ∷ 0 ∷ [])) (ϕ₀-retr a))
+    e-retr-base (zero ∷ one ∷ []) a = cong [_] (cong₂ _+Pℤ/2_
+                                                      (cong (baseP (0 ∷ 1 ∷ [])) (cong fst (ϕ₁-retr _)))
+                                                      (cong (baseP (1 ∷ 0 ∷ [])) (cong snd (ϕ₁-retr _)) ∙ base-neutral _)
+                                               ∙ +Pℤ/2IdR _)
+    e-retr-base (zero ∷ two ∷ []) a = cong [_] (cong (baseP (0 ∷ 2 ∷ [])) (ϕ₂-retr a))
+    e-retr-base (zero ∷ suc (suc (suc m)) ∷ []) a = eq/ _ _ ∣ (v , helper) ∣₁
+      where
+      v = λ { zero → base (0 ∷ m ∷ []) (-ℤ/2 a) ; one → 0Pℤ/2 ; two → 0Pℤ/2 }
+      helper : _
+      helper = +Pℤ/2IdL _ ∙ sym (cong₂ _+Pℤ/2_
+                                       (cong₂ baseP (cong (λ X → 0 ∷ X ∷ []) (+-comm _ _))
+                                       (·ℤ/2IdR _)) (+Pℤ/2IdL _ ∙ +Pℤ/2IdL _) ∙ +Pℤ/2IdR _)
+    e-retr-base (one ∷ zero ∷ []) a = cong [_] (cong₂ _+Pℤ/2_
+                                                      (cong (baseP (0 ∷ 1 ∷ [])) (cong fst (ϕ₁-retr _)) ∙ base-neutral _)
+                                                      (cong (baseP (1 ∷ 0 ∷ [])) (cong snd (ϕ₁-retr _)))
+                                               ∙ +Pℤ/2IdL _)
+    e-retr-base (one ∷ suc m ∷ []) a = eq/ _ _ ∣ (v , helper) ∣₁
+      where
+      v = λ { zero → 0Pℤ/2 ; one → base (0 ∷ m ∷ []) (-ℤ/2 a) ; two → 0Pℤ/2 }
+      helper : _
+      helper = +Pℤ/2IdL _ ∙
+               sym (+Pℤ/2IdL _ ∙ cong₂ _+Pℤ/2_ (cong₂ baseP (cong (λ X → 1 ∷ X ∷ []) (+-comm _ _))
+                                                (·ℤ/2IdR _)) (+Pℤ/2IdR _) ∙ +Pℤ/2IdR _)
+    e-retr-base (suc (suc n) ∷ m ∷ []) a = eq/ _ _ ∣ v , helper ∣₁
+      where
+      v = λ { zero → 0Pℤ/2 ; one → 0Pℤ/2 ; two → base (n ∷ m ∷ []) (-ℤ/2 a) }
+      helper : _
+      helper = +Pℤ/2IdL _ ∙ sym (+Pℤ/2IdL _ ∙ +Pℤ/2IdL _ ∙ +Pℤ/2IdR _
+                                 ∙ cong₂ baseP (cong₂ (λ X Y → X ∷ Y ∷ []) (+-comm _ _) (+-comm _ _))
+                                               (·ℤ/2IdR _))
+
+    e-retr : (x : ℤ/2[x,y]/<y³,xy,x²>) → H*-RP²⋁S¹→ℤ/2[x,y]/<y³,xy,x²> (ℤ/2[x,y]/<y³,xy,x²>→H*-RP²⋁S¹ x) ≡ x
+    e-retr = SQ.elimProp (λ _ → isSetPℤ/2I _ _)
+             (DS-Ind-Prop.f _ _ _ _ (λ _ → isSetPℤ/2I _ _)
+             refl
+             e-retr-base
+             λ {U V} ind-U ind-V → cong₂ _+Pℤ/2I_ ind-U ind-V)
+
+  -----------------------------------------------------------------------------
+  -- Computation of the Cohomology Ring
```

Comment:
Why is all this commented?

Reply:
Waiting for @aljungstrom to prove that, I think he branched the PR